tests/test_custom_extensions_in_hooks.py

```diff
@@ -22,9 +22,11 @@ def template(request):
 
 
 @pytest.fixture
-def output_dir(tmpdir):
+def output_dir(tmp_path):
```

Comment:
We should consider moving this fixture to the base `conftest.py`, to avoid fixture duplication.